### DISCUSSION TITLE
PVP - Unable to differentiate team members in spectate

### DISCUSSION THREAD

**Discussion by Kaillera** [FakeFishGames/Barotrauma#5077] [Ideas and suggestions]
When you're spectating in PVP, you are unable to tell who is on which team. I recommend changing player name colors for spectators to differentiate the players.

Coalition should have blue names
Jovian should have red names

**Comment by HeatedWafflez** [comment]
I'd like to think this is more of an issue than a discussion. Regardless, it's needed nonetheless.
